src/config.rs

```diff
@@ -15,6 +15,22 @@ pub fn default_config_path(mut home_dir: PathBuf) -> String {
     home_dir.to_str().unwrap().to_owned()
 }
 
+#[cfg(not(windows))]
+pub fn global_config_path() -> String {
+    let mut path = PathBuf::from("/etc/puppetlabs/client-tools");
+    path.push("puppetdb");
+    path.set_extension("conf");
+    path.to_str().unwrap().to_owned()
+}
+#[cfg(windows)]
+pub fn global_config_path() -> String {
+    let mut path = PathBuf::from("C:\\ProgramData\\PuppetLabs\\client-tools");
```

Comment:
@briancain @senior Added a conditional compiler flag for using the windows path here. Just fyi.